notebooks/official/model_monitoring_v2/model_monitoring_for_model_outside_vertex.ipynb

```diff
@@ -0,0 +1,1220 @@
+{
```

Comment:
<p>Update the link to Github.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>